week2/2-website/index.html

```diff
@@ -9,13 +10,82 @@
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
 
+    <header class="nav-header">
+        <div class="logo">
+            <img src="/img/karma-logo.svg" alt="karma logo" width=42 height=42>
```

Comment:
```suggestion
            <img src="/img/karma-logo.svg" alt="karma logo" width="42" height="42">
```
all html attributes should be enclosed in double quotes